projects/picker/src/lib/dialog/dialog-ref.class.ts

```diff
@@ -29,20 +29,21 @@ export class OwlDialogRef<T> {
     public componentInstance: T;
 
     /** Whether the user is allowed to close the dialog. */
-    public disableClose = this.container.config.disableClose;
+    public disableClose = true;
```

Comment:
Why this change?

Reply:
Ah, I guess I missed that this is now set to `this.container.config.disableClose` in the constructor instead. Looks good!